tests/transpiler/qasm/test_qasm_to_ionq.py

```diff
@@ -301,15 +323,15 @@ def test_qasm3_to_ionq_zz_native_gate():
             """
     OPENQASM 3.0;
     qubit[2] q;
-    ms(1.1,0) q[0], q[1];
+    ms(1.1,0,0) q[0], q[1];
     """,
             "Invalid phase value",
         ),
         (
             """
     OPENQASM 3.0;
     qubit[2] q;
-    ms(0,-1.5) q[0], q[1];
+    ms(0,-1.5,0) q[0], q[1];
```

Comment:
If OpenQASM supports optional argument to gate definitions, we will want to test both including optional argument and not including it. And if a default value is required, it should be 0.25, not 0.

Reply:
Afaik, OpenQASM does not support optional args to gates. Either we have two gate definitions for ms with distinct names or stick to the max number of args and provide `x` in the arg which is not required such that the term using `x` vanishes.

Reply:
Summarizing our conversation about this and putting on record: I think we decided best approach was to add support for both a two and a three parameter `ms` gate in pyqasm, distinguishing between the two based on the `include` statement e.g. with a custom `ionqnative.inc` to signal `0.25` default